packages/valory/skills/decision_maker_abci/behaviours/decision_receive.py

```diff
@@ -526,7 +526,6 @@ def _update_selected_bet(
             # update the bet's timestamp of processing and its number of bets for the given
             sampled_bet = self.sampled_bet
             sampled_bet.n_bets += 1
-            sampled_bet.processed_timestamp = self.synced_timestamp
```

Comment:
I wonder why we only moved this update into `update_bet_transaction_information` and left the `n_bets` update here. Isn't that inconsistent?

We could remove the decrease in the blacklisting behaviour if that was your concern.

Reply:
yup that was my exact concern, I didn't want to change many bheaviours, will fix it then